cassandra-four-zero/src/main/java/org/apache/cassandra/bridge/CassandraBridgeImplementation.java

```diff
@@ -164,13 +166,31 @@ public static synchronized void setup()
                 throw new RuntimeException(exception);
             }
             config.data_file_directories = new String[]{tempDirectory.toString()};
+            setupCommitLogConfigs(tempDirectory);
             DatabaseDescriptor.setEndpointSnitch(new SimpleSnitch());
             Keyspace.setInitialized();
 
             setup = true;
         }
     }
 
+    public static void setupCommitLogConfigs(Path path)
+    {
+        String commitLogPath = path + "/commitlog";
```

Comment:
can commit logs be disabled for offline tools instead?

Reply:
It should not be necessary. But the current library requires the config to be present. The code path that checks/loads the config is when getting the local host uuid. Seemingly related with newly added CMS. Setting the config is not a big deal, so it is not burning issue unless I missed something. 
1. `org.apache.cassandra.io.sstable.SSTableSimpleWriter#getOrCreateWriter`
2. `org.apache.cassandra.io.sstable.AbstractSSTableSimpleWriter#createWriter`
3. `org.apache.cassandra.io.sstable.SimpleSSTableMultiWriter#create`
4. `org.apache.cassandra.io.sstable.metadata.MetadataCollector#MetadataCollector(org.apache.cassandra.db.ClusteringComparator)`
5. `org.apache.cassandra.service.StorageService#getLocalHostUUID`